README.md

```diff
@@ -308,11 +308,11 @@ NOTE: If you open a link to a Senator's disclosure, you need to paste the URL in
 | Mike Levin | D | CA | House | NO | [2020](https://disclosures-clerk.house.gov/public_disc/financial-pdfs/2020/10040396.pdf) |
 | Ted Lieu | D | CA | House | NO | [2020](https://disclosures-clerk.house.gov/public_disc/financial-pdfs/2020/10042330.pdf) |
 | Zoe Lofgren | D | CA | House | NO | [2020](https://disclosures-clerk.house.gov/public_disc/financial-pdfs/2020/10040456.pdf) |
-| Billy Long | R | MO | House | ? | ? |
-| Barry Loudermilk | R | GA | House | ? | ? |
+| Billy Long | R | MO | House | https://disclosures-clerk.house.gov/public_disc/financial-pdfs/2020/10040351.pdf
```

Comment:
this line has the wrong formatting and the PDF is for matthew cartwright rather than billy long

Reply:
That was sloppy. My apologies.

Reply:
no worries, you're doing great. just thought you might want to edit your pull request - is it clear how to do that? if not, I can just edit it myself.